buildtools/pom.xml

```diff
@@ -170,7 +170,7 @@
               <value>org.apache.pulsar.tests.PulsarTestListener,org.apache.pulsar.tests.AnnotationListener,org.apache.pulsar.tests.FailFastNotifier</value>
             </property>
           </properties>
-          <argLine>
+          <argLine>@{argLine}
```

Comment:
what about using a custom variable name ? argLine is a bit generic

something like
```
<configuration>
  <propertyName>jacoco-surefire-args</propertyName>
</configuration>
```

Reply:
thanks, I set `propertyName` to `testJacocoAgentArgument`. PTAL